posydon/popsyn/normalized_pop_mass.py

```diff
@@ -52,6 +51,8 @@ def initial_total_underlying_mass(df=None, **kwargs):
                                    + sum(initial_ZAMS_mass_2))
     elif isinstance(df, float):
         initial_ZAMS_TOTAL_mass = df
+        initial_ZAMS_TOTAL_single=df1
+        initial_ZAMS_TOTAL_binaries=df2
     else:
```

Comment:
You need to handle `df`, `df1`, and `df2` separately there is no guarantee that they will always have the same type.